Lib/test/test_uuid.py

```diff
@@ -311,6 +311,26 @@ def test_getnode(self):
         node2 = self.uuid.getnode()
         self.assertEqual(node1, node2, '%012x != %012x' % (node1, node2))
 
+    # bpo-32502: UUID1 requires a 48-bit identifier, but hardware identifiers
+    # need not necessarily be 48 bits (e.g., EUI-64).
+    def test_uuid1_eui64(self):
+        # Reset any cached node value
+        self.uuid._node = None
+
+        # Confirm that uuid.getnode ignores hardware addresses larger than 48
+        # bits
+        bad_getter = lambda: 1 << 48
```

Comment:
Why not make this 64 bits?  First, it more closely mirrors the failure in the bug, and second, it eliminates a minor cognitive hiccup.  IOW, you have to think about any off-by-one possibility.  Admittedly, it's minor, but think of the next person reading the code.

Maybe also call the variable `too_large_getter` or some such?

Reply:
I've added some more details in the comment to try to clarify this. `1 << 48`, is, of course, 49 bits and not 48. My goal was to test something just outside the bounds of acceptability and not something well past it.